cmd/internal/ec2/ec2.go

```diff
@@ -0,0 +1,76 @@
+package ec2
+
+import (
+	"fmt"
+
+	"github.com/akitasoftware/akita-cli/cmd/internal/cmderr"
+	"github.com/akitasoftware/akita-cli/rest"
+	"github.com/akitasoftware/akita-cli/telemetry"
+	"github.com/akitasoftware/akita-cli/util"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Mandatory flag: Postman collection id
+	collectionId string
+
+	// Print out the steps that would be taken, but do not do them
+	dryRunFlag bool
+)
+
+var Cmd = &cobra.Command{
+	Use:          "setup",
+	Short:        "Add the Postman Live Collections Agent to EC2.",
+	Long:         "The CLI will add the Postman Live Collections Agent as a systemd service to your EC2 server.",
```

Comment:
Although our purpose is to support EC2 users, and that's what we're documenting, I still worry about making it sound like we do something EC2-specific here.  

What we should try to specify is that this enables LCA on the machine they run the command from!  Not some other server, EC2 or not.

So how about replacing "EC2" with "the current server" or "the local server" (although local might not be easy to understand) or "this server"?

Reply:
Or "current instance" or "this instance" -- to use EC2 terminology?